src/main/java/com/pbl6/dictionaryappbe/persistence/User.java

```diff
@@ -0,0 +1,46 @@
+package com.pbl6.dictionaryappbe.persistence;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.List;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long userId;
+
+    @Column(name = "name", length = 150)
+    private String name;
+
+    @Column(name = "email")
+    private String email;
+
+    @Column(name = "gender", length = 10)
+    private String gender;
+
+    @Column(name = "password")
```

Comment:
Chỗ ni a set thêm value là `columnDefinition = "TEXT"` trong chỗ @Column, mấy chỗ kiểu dữ liệu "TEXT" a thêm cái value này 
Với mấy chỗ nào trong db có set not null thì a để ` nullable = false`